app/routes/admin/tenants.py

```diff
@@ -307,7 +441,7 @@ async def get_tenant(
     return response
 
 
-@router.get("", response_model=List[Tenant])
+@router.get("", response_model=List[Tenant], summary="Get all Tenants")
```

Comment:
Maybe something like: summary="Fetch All Tenants (Paginated)"
and then need query param info added for limit/offset/order_by/descending